src/modules/producer.js

```diff
@@ -56,7 +58,7 @@ class Producer {
       }
 
       // if we found one, we execute the callback and delete it because it will never be received again anyway
-      this._connection.config.transport.info('bmq:producer', `[${queue}] < answer`);
+      this._connection.config.transport.info(loggerAlias, `[${queue}] < answer`);
```

Comment:
line 53, should also have `loggerAlias` IMO

Reply:
I agree.